contrib/kafka/config/500-controller.yaml

```
@@ -0,0 +1,48 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: kafka-controller-manager
+  namespace: knative-sources
+  labels:
+    control-plane: kafka-controller-manager
+spec:
+  selector:
+    matchLabels: &labels
+      control-plane: kafka-controller-manager
+  serviceName: kafka-controller-manager
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      serviceAccountName: kafka-controller-manager
+      containers:
+      - name: manager
+        image: github.com/knative/eventing-sources/contrib/kafka/cmd/controller
+        imagePullPolicy: Always
```

Consider changing it ot 

```suggestion
        imagePullPolicy: IfNotPresent
```

to have minikube tested with local Docker ko.local working?

---

@n3wscott what's the thought here ?

I was thinking about `IfNotPresent` as well ...